### ISSUE TITLE
SCAM iotaseed.io

### ISSUE THREAD

**Issue by Bluephantom3** [norbertvdberg/iotaseed#9]
All funds have been stolen from seeds generated from iotaseed.io

**Comment by redconfetti** [comment]
I don't think the owners of this repo are responsible. Open your browsers code inspector and look at the network tab for http://www.iotaseed.io/ versus http://www.iotaseed.net (compromised)

**Comment by Bluephantom3** [comment]
I did use iotaseed.io not iotaseed.net 

I’m not the only one that has been affected.

**Comment by redconfetti** [comment]
Yeah. I don't have iotaseed.net in my history now that I look at it.

**Comment by rllybadcryptotrdr** [comment]
If you used this site, move your funds. They're all compromised seeds, and they're getting cleaned out as we speak.
